cli/src/commands/install.js

```diff
@@ -58,67 +57,75 @@ export type Args = {
   ignoreDeps?: mixed, // Array<string>
   rootDir?: mixed, // string,
   useCacheUntil?: mixed, // seconds
+  explicitLibDefs: mixed, // Array<string>
 };
 export function setup(yargs: Yargs) {
-  return yargs.usage(`$0 ${name} - ${description}`).options({
-    flowVersion: {
-      alias: 'f',
-      describe:
-        'The Flow version that fetched libdefs must be compatible ' + 'with',
-      type: 'string',
-    },
-    verbose: {
-      describe: 'Print additional, verbose info while installing libdefs',
-      type: 'boolean',
-      demand: false,
-    },
-    skip: {
-      alias: 's',
-      describe: 'Do not generate stubs for missing libdefs',
-      type: 'boolean',
-      demand: false,
-    },
-    libdefDir: {
-      alias: 'l',
-      describe: 'Use a custom directory to install libdefs',
-      type: 'string',
-      demand: false,
-    },
-    cacheDir: {
-      alias: 'c',
-      describe:
-        'Directory (absolute or relative path, ~ is not supported) to store cache of libdefs',
-      type: 'string',
-      demand: false,
-    },
-    packageDir: {
-      alias: 'p',
-      describe: 'The relative path of package.json where flow-bin is installed',
-      type: 'string',
-    },
-    overwrite: {
-      alias: 'o',
-      describe: 'Overwrite an existing libdef',
-      type: 'string',
-      demand: false,
-    },
-    ignoreDeps: {
-      alias: 'i',
-      describe: 'Dependency categories to ignore when installing definitions',
-      choices: ['dev', 'bundled', 'peer'],
-      type: 'array',
-    },
-    rootDir: {
-      alias: 'r',
-      describe: 'Directory of .flowconfig relative to node_modules',
-      type: 'string',
-    },
-    useCacheUntil: {
-      alias: 'u',
-      describe: 'Use cache until specified time in milliseconds',
-      type: 'number',
-    },
-  });
+  return yargs
+    .usage(`$0 ${name} - ${description}`)
+    .positional('explicitLibDefs', {
+      describe: 'Explicitly specify packages to install',
+      default: [],
+    })
+    .options({
+      flowVersion: {
+        alias: 'f',
+        describe:
+          'The Flow version that fetched libdefs must be compatible ' + 'with',
```

Comment:
I know it was already there but wondering about the concatenation... Can't see any use.